<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewUserItem</class>
 <widget class="QDialog" name="NewUserItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>40</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>40</width>
       <height>48</height>
      </size>
     </property>
     <widget class="CRoundLabel" name="image">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>32</y>
        <width>40</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #D197FE, stop:1 #9752ED);
border-radius: 8px;

font-family: &quot;微软雅黑&quot;;
font-size: 11px;
color: #FFFFFF;
</string>
      </property>
      <property name="text">
       <string>在房间</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <property name="topMargin">
      <number>14</number>
     </property>
     <property name="bottomMargin">
      <number>14</number>
     </property>
     <item>
      <widget class="QLabel" name="nameLab">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: &quot;微软雅黑&quot;;
font-weight: 400;
font-size: 16px;
color: #FFFFFF;
line-height: 22px;
font-style: normal;</string>
       </property>
       <property name="text">
        <string>用户昵称占位占位占位占位占…</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font-family: &quot;微软雅黑&quot;;
font-weight: 400;
font-size: 12px;
color: #A8A8A7;
line-height: 17px;
text-align: left;
font-style: normal;</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>24</number>
     </property>
     <property name="leftMargin">
      <number>20</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background: rgba(245,151,255,0.2);
border-radius: 16px;

font-family: &quot;微软雅黑&quot;;
font-weight: 500;
font-size: 16px;
color: #F597FF;
line-height: 22px;
text-align: center;
font-style: normal;
}

QPushButton:checked {
border-radius: 16px;
background: #36383B;

font-family: &quot;微软雅黑&quot;;
font-size: 16px;
color: rgba(255, 255, 255,0.5);

}</string>
       </property>
       <property name="text">
        <string>打招呼</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_3">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
background: #36383B;
border-radius: 16px;

font-family: &quot;微软雅黑&quot;;
font-size: 16px;
color: #ffffff;
}

QPushButton:checked {
background: #36383B;
border-radius: 16px;

font-family: &quot;微软雅黑&quot;;
font-size: 16px;
color: rgba(255,255,255,0.5);

}</string>
       </property>
       <property name="text">
        <string>房间邀请</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="minimumSize">
        <size>
         <width>90</width>
         <height>32</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">

QPushButton {
background: qlineargradient(x1:0, y1:0.5, x2:1, y2:0.5, stop:0 #D197FE, stop:1 #9752ED);
border-radius: 16px;

font-family: &quot;微软雅黑&quot;;
font-size: 16px;
color: #ffffff;
}

QPushButton:checked {
background: #36383B;
border-radius: 16px;

font-family: &quot;微软雅黑&quot;;
font-size: 16px;
color: #ffffff;

}</string>
       </property>
       <property name="text">
        <string>关注</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRoundLabel</class>
   <extends>QLabel</extends>
   <header location="global">CRoundLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
